CalcCurrentValue/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalcCurrentValue {
+    class Program {
+        static void Main(string[] args) {
+
+            Console.WriteLine("#### Input the basic information for the trade below ####");
+            
+            Console.WriteLine("Nominal Amount:");           
+            int nominal = 0;
+            if (!Int32.TryParse(Console.ReadLine(), out nominal)) {
+                Console.WriteLine("Norminal amount need to be an integer number, try again:");
+                nominal = Int32.Parse(Console.ReadLine());
+            }
+
+            Console.WriteLine("Buy or Sell?");          
+            TransactionType BorS =(TransactionType)Enum.Parse(typeof(TransactionType), Console.ReadLine(),true);
```

Comment:
Please try rename variable BorS. It is better to use camelCasing for local variable names and use descriptive names.

Reply:
it stands for 'Buy' or 'Sell' :-)